recipe/meta.yaml

```diff
@@ -13,12 +13,12 @@ source:
 
 build:
   number: 0
-  script: pip install --no-deps .
```

Comment:
See [this issue](https://github.com/conda-forge/staged-recipes/issues/528#issuecomment-216828570) for discussion of pip vs setup.py.

The gist is that pip is really the only official way to install Python packages these days, and anything else is deviating from the norm and should be minimized. Every time a conda-installed package differs in behavior from its pip-installed counterpart is generally considered a bug in the conda package, so keeping the differences in process minimal is desirable from a user and maintenance perspective.

There are two differences between pip and setup.py at this point (that I know of):

1. if a package doesn't use setuptools you can't use the extra arguments, and if it adopts setuptools, you *must* use them to avoid installing an egg. This means that conda recipe maintainers must know whether the package uses setuptools in setup.py or not. pip will install correctly in both cases.
2. For packages with entrypoints, setuptools writes entrypoints that use `pkg_resources`, making setuptools itself a *runtime* dependency, plus dramatically slowing down invocation. pip has fixed this, and creates entrypoints that do not add any dependencies and launch more quickly.

In every known (to me) case where pip and setuptools differ, pip behavior is preferable. Further, any future further deviations between pip and setuptools *should* be picked up by conda packages, so calling pip makes more sense to me than directly invoking setup.py and then trying to keep track of every change in pip and duplicating it.